### DISCUSSION TITLE
Не сопрягается Tion Lite с репозиторием в Home Assistant

### DISCUSSION THREAD

**Discussion by Alexei-vk2** [TionAPI/HA-tion#135] [General]
При сопряжении бризера Tion Lite выдает ошибку: 
Could not pair!
Error occurs while pairing routine! Check logs for details.
Make sure that there is no devices connected to breezer while adding it to Home Assistant.

Mac Адрес вбил верный, но все-равно не подключается.
Как решить проблему? 

When pairing, the Tion Lite breather gives an error: 
"Could not pair!
Error occurs while pairing routine! Check logs for details.
Make sure that there is no devices connected to breeze while adding it to Home Assistant."

I entered the correct Mac address, but it still doesn't connect.
How to solve the problem?

![Снимок экрана 2024-12-09 213122](https://github.com/user-attachments/assets/3fd77405-1ee4-43b8-993b-9edd7b6b27c6)
![Снимок экрана 2024-12-09 213115](https://github.com/user-attachments/assets/12660e2e-d164-442d-810b-0ac42b39ef8b)

**Comment by asan84** [comment]
У меня тоже выдавал ошибку, я вручную сделал:
sudo bluetoothctl и выполнил:
pair MAC-адрес бризера
добавить бризер в HA без сопряжения - галочку убрал

**Reply by Alexei-vk2** [reply]
спасибо за ответ!
я решил вопрос по стабильной работе с помощью ESP32.
Изначально сделал связь по BLE. 
https://github.com/dentra/esphome-tion?ysclid=m6kqvw9b6r951165474


Сейчас вышла прошивка для Tion Lite по UART и сделал по ней:
https://github.com/dentra/esphome-tion/blob/next/configs/tion-3s-uart.yaml

**Reply by ravvor** [reply]
У Вас ошибка в ссылке для tion-lite. Правильную нашел: https://github.com/dentra/esphome-tion/blob/next/configs/tion-lt-uart.yaml
А подскажите, пожалуйста, есть ли рекомендации по "железу"? По подключению, распайке? По остальным бризерам есть в описании, а по Lite пока нету. Или я не нашел?
